src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionResolveHandler.cs

```diff
@@ -23,356 +23,357 @@
 using Roslyn.Utilities;
 using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
 
-namespace Microsoft.CodeAnalysis.LanguageServer.Handler
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler;
```

Comment:
Are the changes in the C# handlers here needed?  If not I'd revert since this PR is already huge

Reply:
Yes, it's needed to support lightbulbs in additional files